### PR TITLE
Add a Spanish translation

### PR DESCRIPTION
**Pull request by Dan-megabyte**
Adding a Spanish translation will help Spicetify reach a greater audience

### CODE REVIEW

File: packages/marketplace/src/resources/locales/es.json [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+      "description": "Una recarga de ventada es necesaria para completar esta operación.",
+      "reloadNow": "Recargar ahora",
+      "reloadLater": "Recargar después"
+    },
+    "devTools": {
+      "title": "Herramientas de desarrollador de temas",
+      "noThemeInstalled": "Error: No tema de marketplace instalado",
+      "noThemeManifest": "Error: No manifiesto de tema encontrado",
+      "colorIniEditor": "Editor de Color.ini",
+      "colorIniEditorPlaceholder": "[your-colour-scheme-name]",
```
Comment (kyrie25) [inline packages/marketplace/src/resources/locales/es.json:50]: This is a sample placeholder value written in English that can be translated.
You want to translate it or keep it in English?

**Review by kyrie25 — COMMENTED** [review]
> Adding a Spanish translation will help Spicetify reach a greater audience

No doubt! I just need to point out a small part here and we should be good to go :)

File: packages/marketplace/src/resources/locales/es.json (hunk 1 above)
Comment (Dan-megabyte) [reply packages/marketplace/src/resources/locales/es.json:50]: As long as it's not user-facing, it should be fine. However, if it is, I'll be happy to replace it to "[nombre-de-esquema-de-color]".

Comment (theRealPadster) [reply packages/marketplace/src/resources/locales/es.json:50]: It is user facing, it's what shows up in the text entry before you've typed anything.

Comment (Dan-megabyte) [reply packages/marketplace/src/resources/locales/es.json]: fixed

**Review by CharlieS1103 — APPROVED** [review]
LGTM

**Review by theRealPadster — COMMENTED** [review]
Awesome thanks! My only other note would be the fact that I have this PR open that adds translations for the import/export feature, and it might make sense for you to add those in here as well. Up to you, can always add in another PR. 
https://github.com/spicetify/spicetify-marketplace/pull/369

**Review by theRealPadster — CHANGES_REQUESTED** [review]
Three PR has been merged in. Could you add the new strings?

**Comment by Dan-megabyte** [comment]
Backup translations has been added.

**Review by theRealPadster — APPROVED** [review]
Awesome thanks!

**Pull request merged**